ptf/saimpls.py

```diff
@@ -3283,8 +3280,7 @@ def mplsEgressPhpTtlModeTest(self):
                 self.client, self.inseg_entry_2222,
                 pop_ttl_mode=SAI_INSEG_ENTRY_POP_TTL_MODE_PIPE)
 
-            # Note: TTL is decremented even though it should not for PHP pipe
-            recv_pkt['IPv6'].hlim = 63
+            recv_pkt['IPv6'].hlim = 62
```

Comment:
Although 62 is correct from a standards point of view, that is not what we implement. This check should be removed.

Reply:
Actually 62 is not correct according to the standards, 64 is correct.